### DISCUSSION TITLE
How do I learn stuff that's not going to be tested?

### DISCUSSION THREAD

**Discussion by giscus** [Tnixc/website-v6#4] [Blog Posts]
# How do I learn stuff that's not going to be tested?

Tnixc's very own website

https://tnixc.space/content/learning.html

**Comment by nuexq** [comment]
I agree focusing in class, grasping the subject, and doing exercises can lead to good marks.. I also study right before exams and still do fine. For learning stuff's that's not going to be tested like:
- Browser Rendering Pipeline
- Networking Protocols
- Events loop

I think that you just need to deep understanding it by dedicating focused time to thoroughly explore them.
